lms/djangoapps/verify_student/views.py

```diff
@@ -1047,3 +1058,142 @@ def reverification_window_expired(_request):
     """
     # TODO need someone to review the copy for this template
     return render_to_response("verify_student/reverification_window_expired.html")
+
+
+class InCourseReverifyView(View):
+    """
+    The in-course reverification view.
+    Needs to perform these functions:
+        - take new face photo
+        - retrieve the old id photo
+        - submit these photos to photo verification service
+
+    Does not need to worry about pricing
+    """
+    @method_decorator(login_required)
+    def get(self, request, course_id, checkpoint_name):
+        """ Display the view for face photo submission
+        """
```

Comment:
Nit: There's no need to put these quotes on their own line; could you move them to the same line as the docstring itself?